apps/www/registry/new-york/example/scroll-area-demo.tsx

```diff
@@ -14,7 +14,9 @@ export default function ScrollAreaDemo() {
         <h4 className="mb-4 text-sm font-medium leading-none">Tags</h4>
         {tags.map((tag) => (
           <>
-            <div className="text-sm">{tag}</div>
+            <div className="text-sm" key={tag}>
```

Comment:
Nit: can we put the key as the first prop for consistency? Thank you.

Reply:
Done @shadcn, thanks.
I moved the prop key as the first prop in New York too.

This are my first commits and PR's here. If something is not ok, let me know :)

Thanks!